libs/chat-workflow/src/main/java/org/finos/springbot/workflow/response/templating/AbstractResourceTemplateProvider.java

```diff
@@ -49,7 +49,7 @@ public F getTemplateForName(String name) {
 		try {
 			return resolveTemplate(name);
 		} catch (Exception e) {
-			LOG.debug("Couldn't find template: "+name);
```

Comment:
I'm not sure it's always an error - remember spring bot will generate a template for you if one doesn't exist